net.brinkervii.grapejuice.yml

```
@@ -0,0 +1,122 @@
+app-id: net.brinkervii.grapejuice 
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: grapejuice
+separate-locales: false
+
+finish-args:
+        - --share=network
+        - --socket=x11
+        - --share=ipc
+        - --socket=wayland
+        - --device=dri
+        - --allow=devel
+        - --socket=pulseaudio
+        - --allow=multiarch
+
+add-extensions:
+        org.freedesktop.Platform.Compat.i386:
+                directory: lib/i386-linux-gnu
+                version: '21.08'
+        org.freedesktop.Platform.Compat.i386.Debug:
+                directory: lib/debug/lib/i386-linux-gnu
+                version: '21.08'
+                no-autodownload: true
+        org.freedesktop.Platform.GL32:
+                directory: lib/i386-linux-gnu/GL
+                version: '1.4'
+                versions: 21.08;1.4
+                subdirectories: true
+                no-autodownload: true
+                autodelete: false
+                add-ld-path: lib
+                merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+                download-if: active-gl-driver
+                enable-if: active-gl-driver
+        org.freedesktop.Platform.VAAPI.Intel.i386:
+                directory: lib/i386-linux-gnu/dri/intel-vaapi-driver
+                version: '21.08'
+                subdirectories: true
+                autodelete: false
+                add-ld-path: lib
+                download-if: active-gl-driver
+                enable-if: active-gl-driver
+modules:
+  - python3-requirements.yml
+  - name: bundle-setup
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib32
+      - mkdir -p /app/lib/i386-linux-gnu
+      - mkdir -p /app/lib/debug/lib/i386-linux-gnu
+      - mkdir -p /app/lib/i386-linux-gnu/GL
+      - install -Dm644 ld.so.conf /app/etc/ld.so.conf
+    sources:
+      - type: file
+        dest-filename: ld.so.conf
+        url: data:/app/lib32%0A/app/lib/i386-linux-gnu%0A
+  - name: grapejuice
+    buildsystem: simple
+    build-commands:
+      # install usersettings
+      - install -Dm755 user_settings.json /app
```

```suggestion
      - install -Dm644 user_settings.json /app
```

This is not an executable.